jwst/jump/twopoint_difference.py

```diff
@@ -9,31 +9,30 @@
 outliers and set the appropriate DQ mask for all outliers in the pixel.
 
 This is MUCH faster than doing all the work on a pixel-by-pixel basis.
-'''
+"""
 
 import logging
 import numpy as np
 
-from ..datamodels import dqflags
-from .. import datamodels
+import dqflags
```

Comment:
For use in the package structure this will need to be:

from ..datamodels import dqflags

so that the dqflags module is imported from the correct location within the package.